deploy.yml

```
@@ -0,0 +1,38 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "main"
+  main:
+    name: Build, Validate and Deploy
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the spec-prod (https://w3c.github.io/spec-prod/)
+      # the document is built and validated as a check in the pull request.
+      # the document is built and validated, and then deployed to gh-pages branch, when a commit is pushed to the main branch.
+      - uses: w3c/spec-prod@v2
+        with:
+          TOOLCHAIN: bikeshed
+          VALIDATE_MARKUP: true
+          GH_PAGES_BRANCH: gh-pages
+          DESTINATION: specification/index.html
+          # Deployment will be available at: https://<org>.github.io/<repo>/specification/
```

```suggestion
          # Deployment will be available at: https://w3c.github.io/webextensions/specification/
          DESTINATION: specification/index.html
```

Move the comment above the output destination and make the URL specific to our repo.